src/bot/__main__.py

```diff
@@ -1,15 +1,18 @@
+import discord
+from discord.ext import commands
+
 from glob import glob
 from pathlib import Path
 from constants import environment
+from discord_slash import SlashCommand
 
 from arena_bot import ArenaBot
 
-
-bot = ArenaBot(command_prefix="!")
-
+bot = commands.Bot(command_prefix="!")
+slash = SlashCommand(bot, override_type=True)
```

Comment:
What is `override_type`?
Why had it been set to true?

Reply:
We are overriding the bot prefix because we need to set a bot prefix to run the bot.